lib/common/CommonPal.h

```diff
@@ -202,10 +195,23 @@ PALIMPORT PSLIST_ENTRY PALAPI InterlockedPopEntrySList(IN OUT PSLIST_HEADER List
 
 #endif // _WIN32
 
+
+// Use intsafe.h for internal builds (currently missing some files with stdint.h)
+#if defined(_WIN32) && defined(NTBUILD)
+#define ENABLE_INTSAFE_SIGNED_FUNCTIONS 1
+#include<intsafe.h>
+#else
+#include<stdint.h>
```

Comment:
> Shouldn't these have spaces like #include <stdint.h> to be consistent with the rest of includes in the project?

Yes, thanks for catching! I'll correct them in next change (cleanup) to save CI some cycles.